1. Developer basics/6_ides/6-2_intellij_vscode_configuration.md

```diff
@@ -0,0 +1,145 @@
+- `Ctrl+Enter` (macOS: `Cmd+Enter`) in the Quick Open (`Ctrl+P`) file list.
+
+### [Editor Groups](#https://code.visualstudio.com/docs/getstarted/userinterface#_editor-groups)
+
+When you split an editor (using the **Split Editor** or **Open to the Side** commands), a new editor region is created which can hold a group of items. You can open as many editor regions as you like side by side vertically and horizontally.
+
+### [Grid editor layout](https://code.visualstudio.com/docs/getstarted/userinterface#_grid-editor-layout)
+
+By default, editor groups are laid out in vertical columns (for example when you split an editor to open it to the side). You can easily arrange editor groups in any layout you like, both vertically and horizontally:
+
+There are a predefined set of editor layouts in the new **View** > **Editor Layout** menu. Editors that open to the side will by default open to the right-hand side of the active editor. If you prefer to open editors below the active one, configure the new setting `workbench.editor.openSideBySideDirection: down`.
+
+### [Color Themes](#https://code.visualstudio.com/docs/getstarted/themes)
+
+Color themes let you modify the colors in Visual Studio Code's user interface to suit your preferences and work environment.
+
+### [Extension](#https://code.visualstudio.com/docs/editor/extension-marketplace)
+
+The features that Visual Studio Code includes out-of-the-box are just the start. VS Code extensions let you add languages, debuggers, and tools to your installation to support your development workflow.
+
+You can browse and install extensions from within VS Code. Bring up the Extensions view by clicking on the Extensions icon in the **Activity Bar** on the side of VS Code or the **View: Extensions** command (`Ctrl+Shift+X`). To install an extension, select the **Install** button. Once the installation is complete, the **Install** button will change to the **Manage** gear button.
+
+### [Debugging/Run](#https://code.visualstudio.com/docs/editor/debugging)
+
+To run or debug a simple app in VS Code, select Run and Debug on the Debug start view or press F5 and VS Code will try to run your currently active file.
+
+However, for most debugging scenarios, creating a launch configuration file is beneficial because it allows you to configure and save debugging setup details. VS Code keeps debugging configuration information in a `launch.json` file located in a `.vscode` folder in your workspace (project root folder) or in your [user settings](#https://code.visualstudio.com/docs/editor/debugging#_global-launch-configuration) or [workspace settings](#https://code.visualstudio.com/docs/editor/multi-root-workspaces#_workspace-launch-configurations).
+
+To create a `launch.json` file, click the create a launch.json file link in the Run start view. VS Code will try to automatically detect your debug environment, but if this fails, you will have to choose it manually.
+
+### Linter
```

Comment:
Markdown lint (Include a new line after the title)

Reply:
Thanks for the suggestion, @jase156 there is an excellent extension for [vscode](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) that help us linting markdown files.